web/frontend/src/language/fr.json

```
@@ -75,49 +75,49 @@
     "addUser": "Ajouter un utilisateur",
     "role": "Rôle",
     "roles": "Rôles",
-    "edit": "Editer",
-    "nothingToAdd": "Il y a rien à ajouter.",
+    "edit": "Éditer",
+    "nothingToAdd": "Il n'y a rien à ajouter.",
     "duplicateCandidate": "Ce candidat a déjà été ajouté.",
     "add": "Ajouter",
     "exportJSON": "Exporter en JSON",
     "delete": "Supprimer",
-    "addVoters": "Ajouter électeur·rice·s",
+    "addVoters": "Ajouter des électeur·rice·s",
     "combineShares": "Combiner les actions",
     "createElec": "Créer un sondage",
     "clearForm": "Effacer un sondage",
     "elecName": "Titre du sondage",
-    "confirmRemovesubject": "Voulez vous vraiment supprimer ce sujet?",
-    "confirmRemovetext": "Voulez vous vraiment supprimer ce texte?",
-    "confirmRemoverank": "Voulez vous vraiment supprimer ce rang?",
-    "confirmRemoveselect": "Voulez vous vraiment supprimer cette selection?",
-    "upload": "Choisi un fichier json sur votre ordinateur:",
+    "confirmRemovesubject": "Voulez-vous vraiment supprimer ce sujet?",
+    "confirmRemovetext": "Voulez-vous vraiment supprimer ce texte?",
+    "confirmRemoverank": "Voulez-vous vraiment supprimer ce rang?",
+    "confirmRemoveselect": "Voulez-vous vraiment supprimer cette sélection?",
+    "upload": "Choisissez un fichier json sur votre ordinateur:",
```

```suggestion
    "upload": "Choisissez un fichier JSON sur votre ordinateur:",
```
so that we're coherent w/ ln. 97

---

I noticed it too. But I thought I'd do another PR where we'd homogenize the JSON and SCIPER names in all en.json, fr.json and de.json.